src/GoogleAuthenticator.php

```diff
@@ -136,7 +126,7 @@ public function generateSecret()
      *
      * @return int
      */
-    protected static function hashToInt($bytes, $start)
+    private function hashToInt(string $bytes, int $start): int
```

Comment:
I think we should also change docblock `@param` section also if we doing this typehint change